src/renderer/components/item-object-list/item-list-layout.tsx

```diff
@@ -123,12 +121,6 @@ export class ItemListLayout extends React.Component<ItemListLayoutProps> {
 
     if (preloadStores) {
       this.loadStores();
-
-      if (!isClusterScoped) {
-        disposeOnUnmount(this, [
-          namespaceStore.onContextChange(() => this.loadStores())
-        ]);
-      }
```

Comment:
This is no longer needed `NamespaceStore` as loading stores is already responsive to this.